### PR TITLE
updated references of base plugins to netcommon

### PR DESCRIPTION
**Pull request by KB-perByte**
##### SUMMARY

Minimum required ansible.netcommon version is 2.5.1
Updated base plugin references to ansible.netcommon

##### ISSUE TYPE

- Feature Pull Request

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

**Comment by NilashishC** [comment]
recheck

File: galaxy.yml [hunk 1]
```diff
@@ -2,7 +2,7 @@
 authors:
   - Ansible Network Community (ansible-network)
 dependencies:
-  "ansible.netcommon": ">=2.0.1"
+  "ansible.netcommon": ">=2.5.0"
```
Comment (Qalthos) [inline galaxy.yml:5]: This has to be 2.5.1 to work
```suggestion
  "ansible.netcommon": ">=2.5.1"
```

**Review by Qalthos — CHANGES_REQUESTED** [review]

File: changelogs/fragments/netcommon_ref_update.yaml [hunk 2]
```diff
@@ -0,0 +1,4 @@
+---
+major_changes:
+  - "Minimum required ansible.netcommon version is 2.5.0."
```
Comment (Qalthos) [inline changelogs/fragments/netcommon_ref_update.yaml:3]:
```suggestion
  - "Minimum required ansible.netcommon version is 2.5.1."
```

**Comment by NilashishC** [comment]
recheck

**Review by Qalthos — APPROVED** [review]

**Comment by NilashishC** [comment]
recheck

**Comment by NilashishC** [comment]
recheck

**Review by NilashishC — APPROVED** [review]

**Comment by NilashishC** [comment]
recheck

**Pull request merged**